quinn-interop/src/client.rs

```diff
@@ -113,6 +113,26 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     };
     let config = config("hq-interop", suites)?;
 
+    if testcase == "multiconnect" {
+        let mut set = JoinSet::new();
+        for request in requests {
+            let config = config.clone();
+            set.spawn(async move {
+                let result = async {
+                    let (_, conn) = connect(&request, config.clone()).await?;
+                    hq_download(conn, request).await?;
+                    Ok::<(), Box<dyn std::error::Error>>(())
+                }
+                .await;
+                if let Err(e) = result {
+                    error!("request failed: {}", e);
+                }
+            });
+        }
+        while let Some(_) = set.join_next().await {}
+        return Ok(());
```

Comment:
Shouldn't this be able to yield `Err` somehow in order to propagate test case failure?